src/index.js

```diff
@@ -0,0 +1,11 @@
+import React from 'react';
+import ReactDOM from 'react-dom/client';
+import './index.css';
+import Calculator from './components/calculator';
+
+const root = ReactDOM.createRoot(document.getElementById('root'));
+root.render(
+  <React.StrictMode>
+    <Calculator />
+  </React.StrictMode>,
+);
```

Comment:
It's good that you have removed the boilerplate files by the CRA. By convention, App.js and App.css make the main component and therefore should not be deleted, only the content that is not necessary should be removed. Kindly include them.

Reply:
I have added both files to the project.